parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1831,8 +1828,8 @@ toSlurpResult currentPath uf existingNames =
   fileNames0 = UF.typecheckedToNames0 uf
 
   sc :: R.Relation Name Referent -> R.Relation Name Reference -> SlurpComponent v
-  sc terms types = SlurpComponent { terms = Set.map var (R.dom terms)
-                                  , types = Set.map var (R.dom types) }
+  sc terms types = SlurpComponent { terms = Set.map Name.toVar (R.dom terms)
+                                  , types = Set.map Name.toVar (R.dom types) }
```

Comment:
Same here: maybe we can skip `Name`, but we'd have to dig into the rest of the function.

Reply:
Maybe we would benefit from using `Var v => Names' v` rather than `Names' Name` aka `Names2.Names0` in these functions; it's never been done before!  I don't think `Names'` existed when this code was written.